make/targets/golang/imagebuilder.mk

```diff
@@ -0,0 +1,30 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+
+IMAGEBUILDER_VERSION ?=1.2.1
+
+IMAGEBUILDER ?= $(shell which imagebuilder 2>/dev/null)
+ifneq "" "$(IMAGEBUILDER)"
+_imagebuilder_installed_version = $(shell $(IMAGEBUILDER) --version)
+endif
+
+# NOTE: We would like to
+#     go get github.com/openshift/imagebuilder/cmd/imagebuilder@v$(IMAGEBUILDER_VERSION)
+# ...but `go get` is too unreliable. So instead we use this to make the
+# "you don't have imagebuilder" error useful.
+ensure-imagebuilder:
+ifeq "" "$(IMAGEBUILDER)"
+	$(error Could not find required tool "imagebuilder". Try installing it via: go get github.com/openshift/imagebuilder/cmd/imagebuilder@v$(IMAGEBUILDER_VERSION))
```

Comment:
imagebuilder not found! Get it by `go get github.com/openshift/imagebuilder/cmd/imagebuilder@v$(IMAGEBUILDER_VERSION))`

Reply:
Sorry, are you suggesting that I reword the message to make it shorter?

Reply:
Yes, please.

Reply:
Ack, done. (s/by/with/ for grammatical correctness)